app/views/index.scala.html

```diff
@@ -199,23 +199,23 @@
             <section class="collaborator-images-container">
                 <div class="collaborator-image">
                     <a id="makeabilitylab-img-link" href="https://makeabilitylab.cs.washington.edu/">
-                        <img src='@routes.Assets.at("assets/MakeabilityLogo.png")' alt='Makeability_Logo'>
+                        <img src='@routes.Assets.at("assets/MakeabilityLogo.png")' alt='Makeability Lab Logo'>
```

Comment:
You should just use the actual translations here. I know that you did that in your other PR, but you've ultimately set it up so that I have to merge both of your PRs at once, otherwise neither can be merged. 

You should make each PR as separate from each other as possible, and we can deal with merge conflicts in the future if it ends up being an issue.